packages/core/src/getNextSnapshot.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  AnyActorLogic,
+  AnyActorScope,
+  EventFromLogic,
+  SnapshotFrom,
+  createEmptyActor
+} from '.';
```

Comment:
we should avoid circular deps - let's import those from places in which they originate and not from the `index.ts`